action/candidate_update.go

```diff
@@ -111,6 +111,10 @@ func NewCandidateUpdate(
 			return nil, err
 		}
 	}
+
```

Comment:
1. remove empty line
2. can you check if this would cause a potential different result in `workingset.pickAndRunActions()`? If yes, we need to gate the code by HF

Reply:
pickAndRunActions

> 1. remove empty line
> 2. can you check if this would cause a potential different result in `workingset.pickAndRunActions()`? If yes, we need to gate the code by HF

No difference,  invalidCandName actions will be discarded